app/main.py

```diff
@@ -54,5 +42,16 @@ def value_set(
         "identifier": identifier,
```

Comment:
can you add `code` and `version` params here? (I'm looking at sec. 5.1.2 and 5.1.3 for reference)

Reply:
Added, good catch!